server/graphql/v2/input/ConnectedAccountCreateInput.ts

```diff
@@ -0,0 +1,22 @@
+import { GraphQLString, GraphQLInputObjectType } from 'graphql';
+import GraphQLJSON from 'graphql-type-json';
+import { ConnectedAccountService } from '../enum/ConnectedAccountService';
+
+/**
+ * An input for ConnectedAccount that can be used for either editing or creating.
+ */
+const ConnectedAccountCreateInput = new GraphQLInputObjectType({
+  name: 'ConnectedAccountCreateInput',
+  fields: {
+    clientId: { type: GraphQLString },
+    data: { type: GraphQLJSON },
+    id: { type: GraphQLString },
+    refreshToken: { type: GraphQLString },
+    settings: { type: GraphQLJSON },
+    token: { type: GraphQLString },
+    service: { type: ConnectedAccountService },
+    username: { type: GraphQLString },
```

Comment:
Let's add a description of all this fields, and maybe remove the one that might not be needed as user input.